bugshaker/src/main/java/com/github/stkent/bugshaker/flow/email/FeedbackEmailFlowManager.java

```diff
@@ -205,8 +219,14 @@ public void startFlowIfNeeded(Context context,
 		}
 
 		this.ignoreFlagSecure = ignoreFlagSecure;
+		ActivityManager activityManager = (ActivityManager)context.getSystemService(Context.ACTIVITY_SERVICE);
+		ComponentName componentName = activityManager.getRunningTasks(1).get(0).topActivity;
+
+		String str1 = componentName.toString();
+		String str2 = "ComponentInfo{com.github.stkent.bugshaker/com.github.stkent.bugshaker.MainActivity}";
```

Comment:
Originally we had this check in place because we wanted to avoid detecting shakes from the annotation activity, but it didn't work so we just got rid of this code